data/json/professions.json

```diff
@@ -7773,6 +7869,33 @@
       "female": { "entries": [ { "item": "panties" }, { "item": "bra" } ] }
     }
   },
+  {
```

Comment:
I don't see much of a difference between `ittech` and `programmer` conceptually. If I want to play as a computer guy, then the programmer is more cool, since they have SD card and a USB stick.

Reply:
I did consider it a bloat, but I have played with only a handful of classes. You do you. :)